.travis.yml

```diff
@@ -24,6 +24,7 @@ script:
   - ls -la bin/linux/amd64
   - amp --server=localhost cluster create --tag=local --registration=none --notifications=false
   - TESTINCLUDE=platform/testing testrunner platform/tests
+  - tests/run
```

Comment:
Some of the tests are accessing services that are not exposed outside of our swarm (`etcd` or `nats` for instance). These are the options I see:
 - Create an integration test container (that's what I did),
 - Expose these ports externally for tests purposes,
 - Mount the source code of `amp` in the `DinD` cluster and use this for tests.
 - Use a local swarm instead of a `DinD` swarm.